api/src/lib/serverless.ts

```diff
@@ -0,0 +1,3 @@
+import type {AWS} from '@serverless/typescript'
+
+export type FunctionDefinition = AWS['functions'][string]
```

Comment:
Serverless types really suck because they define them all in one blob rather than individually, so by pulling this type out, we allow splitting parts of this object out without losing type hints in the second location.